test/bdd/bddtests_test.go

```diff
@@ -108,6 +108,8 @@ func getCmdArg(argName string) string {
 
 func FeatureContext(s *godog.Suite) {
 
+	log.SetLevel("", log.DEBUG)
```

Comment:
@fqutishat aren't we passing log levels in?

Reply:
Reverted this line, enabled it to see logs locally.